mteb/evaluation/MTEB.py

```diff
@@ -463,6 +466,29 @@ def run(
                 f"\n\n********************** Evaluating {task.metadata.name} **********************"
             )
 
+            if isinstance(task, AbsTaskAggregate):
```

Comment:
Recursively resolving Aggregated tasks allows arbitrary nesting of AbsTaskAggregate.